airbyte-migration/src/main/resources/migrations/migrationV0_14_3/airbyte_config/StandardSync.yaml

```diff
@@ -11,6 +11,9 @@ required:
   - catalog
 additionalProperties: false
 properties:
+  prefix:
```

Comment:
Why is this file changing?

Reply:
We didn't introduce this change back when we wrote this migration. Thats why its required